Application/Setting/Consumer/Service/Data.php

```diff
@@ -307,21 +322,26 @@ public function setupDatabaseContent()
                 'Bei der alphabetischen Sortierung von Namen werden Präpositionen (z.B. von, de, etc.) für die Sortierung berücksichtigt [Standard: Ja]',
                 true);
         }
+
+        // Fehlzeiten
+        $this->createSetting('Education', 'ClassRegister', 'Absence', 'UseClassRegisterForAbsence', TblSetting::TYPE_BOOLEAN,
+            '1', 'Fehlzeiten', 'Automatische Übernahme der Fehlzeiten aus dem Klassenbuch [Standard: Ja]'
+            , true);
     }
 
     /**
-     * @param      $Cluster
-     * @param      $Application
-     * @param null $Module
-     * @param      $Identifier
+     * @param        $Cluster
+     * @param        $Application
+     * @param null   $Module
+     * @param string $Identifier
      *
      * @return false|TblSetting
      */
     public function getSetting(
         $Cluster,
         $Application,
         $Module = null,
-        $Identifier
+        $Identifier = ''
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$Identifier" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)